packages/base/adapters/solana/web3js/providers/WalletStandardProvider.ts

```diff
@@ -65,17 +65,16 @@ export class WalletStandardProvider extends ProviderEventEmitter implements Prov
 
   // -- Public ------------------------------------------- //
   public get name() {
+    if (this.wallet.name === 'Trust') {
+      // The wallets from our list of wallets have not matching with the extension name
+      return 'Trust Wallet'
+    }
+
     return this.wallet.name
   }
 
   public get type() {
-    const FILTER_OUT_ADAPTERS = ['Trust']
-
-    if (FILTER_OUT_ADAPTERS.includes(this.wallet.name)) {
-      return 'EXTERNAL'
-    }
-
-    return 'ANNOUNCED'
+    return 'ANNOUNCED' as const
   }
```

Comment:
why remov this?

Reply:
I think this was added in an attempt to fix duplicated Trust wallets, which now I've changed the fix in the lines above. And this was making Trust appear without "Installed" tag like in the image 
![image](https://github.com/user-attachments/assets/9d613c4c-1679-425c-99b3-a0ecca13319c)